.spelling

```diff
@@ -863,6 +868,7 @@ RPC
 RPCs
 runtime
 runtimes
+RuntimeClass
```

Comment:
RuntimeClass should likely be backpacked and not added here.

Reply:
Hi @ericvn okay, done!